igvc_ws/src/sensors/src/compass_node.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+
+
+'''
+    Description: Compass node
+    Date Modified: 1 Feb 2019
+    Author: Ben Brenkman
+'''
+
+import ros_api as ros
+from ros_api import println
+import rospy
```

Comment:
You don't need to import rospy, ropspy.spin is implemented in the ROS_API, just call ros.spin() on line 35.

Reply:
You need to take a look at the API. It didn't work when I tried it.

Reply:
Nevermind, I found the problem